Tests/SentryProfilerTests/SentryProfileTestFixture.swift

```diff
@@ -142,34 +182,34 @@ class SentryProfileTestFixture {
         func renderGPUFrame(_ type: GPUFrame) {
             switch type {
             case .normal:
-                let currentSystemTime: UInt64 = currentDateProvider.systemTime()
-                print("expect normal frame to start at \(currentSystemTime)")
+                let timestamp = String(currentDateProvider.systemTime())
+                log(#line, "expect normal frame to start at \(timestamp)")
```

Comment:
`m`: What's the benefit of logging all that information? Do we really need that? This is a lot of information. This refers to all log messages here.

Reply:
it helped debug things, but moreso when i was still trying to get the simulation working. I needed to know where things actually went wrong before the eventual failed test assertion.

I typically find that helpful logs are much too voluminous to read line by line for understanding, but it's helpful to have the info logged if you know what to search for. In projects I've worked on in the past that have both more voluminous and mature logging solutions, they had a way to set log levels per component. So when you're debugging, you could set only replays/network/tests to debug level, and everything else to info/warn, to focus on just the components you're working on. 

I thought I'd already filed an issue for this, but couldn't find anything in GitHub or even Notion, so I created one: https://github.com/getsentry/sentry-cocoa/issues/4020